modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ImportAndDeleteObjectEntryPerformanceTest.java

```diff
@@ -56,11 +63,42 @@ public static void assume() {
 
 	@Test
 	public void testImportAndDeleteObjectEntry() throws Exception {
+		_addCompany();
 		_importObjectFolder();
 		_importObjectEntry();
 		_deleteObjectEntry();
 	}
 
+	private void _addCompany() throws Exception {
```

Comment:
@slnn Please use CompanyTestUtil.addCompany();

Reply:
Hi @tinatian 

We can not use these and addcompany() methods in CompanyTestUtil directly because these addComany() methods will use a random string as the virtual host of the company,  it is OK to use them in an integration test which will call the API of portal directly instead of call the REST API by **HttpInvoker**, when using the **HttpInvoker** to call the REST API in a non-default company, it requires that we need to know the  the virtual host of the company in advance, because the URL that visited the REST API will be changed from http://localhost:8080/o/c/foos/batch to  http://VIRTAUL-HOST-NAME:8080/o/c/foos/batch  for example, in current change, the virtual host of the non-default company is www.able.com, if we want to run the test locally, we need to set `127.0.0.1	www.able.com` in the hosts then running the test, otherwise, **HttpInvoker** will complain NoSuchHostException.

Reply:
@slnn @tinatian It seems `UserAccountResourcePerformanceTest` and other generated `*ResourceTest` are not creating a new company?